docs/functions-develop-schema-registry.md

```diff
@@ -4,6 +4,19 @@ title: Use schema registry
 sidebar_label: "Use schema registry"
 ---
 
-Pulsar has a built-in schema registry and is bundled with popular schema types, such as Avro, JSON and Protobuf. Pulsar Functions can leverage the existing schema information from input topics and derive the input type. The schema registry applies to output topics as well.
+Pulsar has a built-in [schema registry](schema-overview.md) and is bundled with popular [schema types](schema-understand.md#schema-type). Pulsar Functions can leverage the existing schema information from input topics and derive the input type. The schema registry applies to output topics as well.
 
-For more details, refer to [code example](https://github.com/apache/pulsar/blob/master/pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/AutoSchemaFunction.java).
+The following table outlines the supportability of schema types in Pulsar Functions.
+
+| Schema Type    | Java Function | Python Function | Go Function |
+|----------------|---------------|-----------------|-------------|
+| String         | ✅             | ✅               |             |
```

Comment:
```suggestion
| String         | ✅             | ✅               | Not supported            |
```

Suggest indicates "not supported" (or use ❌ to keep consistent) explicitly to reduce users' guesswork. 

(In this case, "Not supported" or "❌" is not a negative signal, it just shows the status and fact)

Reply:
I understand your concern, but I don't think it will make readers guess without those inverted indicators; on the contrary, IMO, adding "not supported" is a bit redundant. What's worse, to some users, ❌ also means "selected" which leads to confusion.

Googled a few more references from MS and Splunk:
<img width="501" alt="image" src="https://user-images.githubusercontent.com/60642177/227077830-40609dd3-7878-4b17-8e46-8169c7c2f249.png">
<img width="638" alt="image" src="https://user-images.githubusercontent.com/60642177/227077867-2dd294c6-7f45-4912-a341-99d4f28afd58.png">

Reply:
If there is no ✅, users (like German) might take ❌ as "supported". Since there is ✅, why do users still take ❌ as "supported"?

Reply:
I got the feedback from a customer-facing team when both indicators were placed in a table years ago. They felt confused because they weighed X higher than any others to represent YES in a table.
So, taking this into considerations, I don't see the need for adding inverted ones here. It's clear and specific enough for users to read the table.

Reply:
From my past experience, some users asked me what the meaning of "empty"? They take "empty" as "maintainers **forget** to fill info" and "**unsure** whether it means unsupported or not" because **explicit** answers are not given — this is the [guesswork](https://github.com/apache/pulsar-site/pull/486#discussion_r1145530721) that happened among users and should be avoided from a rigorous perspective.

Reply:
Thanks for sharing more ideas! 
I can understand the guesswork of an empty cell that you mentioned when it comes to a parameter table in terms of name, description, default value, etc. The case in this PR is different, which is all about a Yes/No answer, without other options. 

Regarding this example https://github.com/apache/pulsar-site/pull/486#discussion_r1145775435, I think it's ambiguous. I guess the meaning of the hollow circle sign is "partially supported" but I'm not sure.

I will merge the PR and revisit this thread if I find a compromised way to balance different user experiences :)